include/gatekeeper_gt.h

```diff
@@ -202,6 +203,11 @@ struct gt_cmd_entry {
 
 	union {
 		lua_State *lua_state; /* GT_UPDATE_POLICY */
+		/* GT_UPDATE_POLICY_INCREMENTALLY */
+		struct {
+			size_t len;
+			char lua_bytecode[128];
```

Comment:
Instead of a file name, we should pass a pointer to a memory block with the byte code.